addons/editor-devtools/DevTools.js

```diff
@@ -38,7 +38,7 @@ export default class DevTools {
   async init() {
     this.addContextMenus();
     while (true) {
-      const root = await this.addon.tab.waitForElement("ul[class*=gui_tab-list_]", {
+      const root = await this.addon.tab.waitForElement("div[class^=gui_tabs_] ul[class*=gui_tab-list_]", {
```

Comment:
Does this change anything in the current state of the PR?

Reply:
i think it's so it won't select the tabs in the debugger modal

Reply:
@GarboMuffin Can you answer this?

Reply:
This originally fixed a debugger & devtools compatibility issue. I believe this is now unnecessary with mxmou's tab changes